### PR TITLE
Apply new nuget packages

### PR DESCRIPTION
**Pull request by Sanan07**
## Description
Add in TP new nuget packages

## Related issue
Kindly link any related issues. E.g. Fixes #xyz.

### CODE REVIEW

File: src/package/nuspec/Microsoft.TestPlatform.nuspec [hunk 1]
```diff
@@ -129,20 +129,6 @@
     <file src="net451\$Runtime$\Microsoft.VisualStudio.QualityTools.LoadTest.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.QualityTools.LoadTest.dll" />
     <file src="net451\$Runtime$\Microsoft.VisualStudio.QualityTools.LoadTestFramework.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.QualityTools.LoadTestFramework.dll" />
     <file src="net451\$Runtime$\Microsoft.VisualStudio.QualityTools.Resource.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.QualityTools.Resource.dll" />
-    <file src="net451\$Runtime$\Microsoft.VisualStudio.QualityTools.Tips.GenericTest.Adapter.dll"	target="tools\net451\Common7\IDE\Extensions\TestPlatform\Microsoft.VisualStudio.QualityTools.Tips.GenericTest.Adapter.dll" />
```
Comment (Haplois) [inline src/package/nuspec/Microsoft.TestPlatform.nuspec:132]: I assume this is intended. Files are removed but not replaced.

File: test/TestAssets/MstestV1UnitTestProject/MixedTestsRelease.orderedtest [hunk 2]
```diff
@@ -1,9 +1,5 @@
...
-    <TestLink id="4e29cabf-6b42-433d-43ac-55a67fe11ab2" name="PassingTest2" storage="bin\release\net451\MstestV1UnitTestProject.dll" type="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestElement, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.ObjectModel, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
-	<TestLink id="26da5e66-905a-30e9-2bd8-754df1300777" name="FailingTest1" storage="bin\release\net451\MstestV1UnitTestProject.dll" type="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestElement, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.ObjectModel, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
-    <TestLink id="080755a1-efc5-3620-f352-0c0c9d220cf3" name="FailingTest2" storage="bin\release\net451\MstestV1UnitTestProject.dll" type="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestElement, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.ObjectModel, Version=12.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" />
+  <TestLinks>
```
Comment (Haplois) [inline test/TestAssets/MstestV1UnitTestProject/MixedTestsRelease.orderedtest:3]: No tests in the file, double-check it, please.

File: src/package/sign/sign.proj [hunk 3]
```diff
@@ -91,9 +91,12 @@
       <AssembliesToSign Include="$(ArtifactsDirectory)Extensions\Microsoft.VisualStudio.TestTools.CppUnitTestFramework.CppUnitTestExtension.dll" />
 
       <!-- UWP runtime provider and dependent dlls -->
-      <AssembliesToSign Include="$(ArtifactsDirectory)CoreCon\VisualStudioDatastoreConfigurationProvider.dll" />
+      <!-- We removed this dll from package-->
+      <!--<AssembliesToSign Include="$(ArtifactsDirectory)CoreCon\VisualStudioDatastoreConfigurationProvider.dll" />-->
```
Comment (Haplois) [inline src/package/sign/sign.proj:95]: If it's removed, please delete it, instead of commenting it.

Comment (Sanan07) [reply src/package/sign/sign.proj:95]: It is removed, but we will send the other team our nuget package to test without that dll. If it does not work will put this dll back. So will update comment itself.

**Review by Haplois — APPROVED** [review]

**Pull request merged**